app/pages/docs/get-started.mdx

```diff
@@ -1,82 +1,84 @@
 ---
-title: Get Started with Blitz
-sidebar_label: Get Started
+title: Comece a Usar Blitz
+sidebar_label: Comece a Usar
 ---
 
-### Set Up Your Computer {#set-up-your-computer}
+### Configure Seu Computador {#set-up-your-computer}
 
-You need Node.js 12 or newer. You can verify this by running `node -v` in
-your terminal. If you don't have Node or need a newer version, we
-recommend using a node version manager like
-[fnm](https://github.com/Schniz/fnm). That will allow you to change node
-versions and even have different versions for each project.
+Você precisa do Node.js na versão 12 ou mais recente. Você pode verificar
+isso rodando `node -v` no seu terminal. Se você não tem o Node instalado
+ou precisa de uma versão mais recente, nós recomendamos usar um
+gerenciador de versões de Node como [fnm](https://github.com/Schniz/fnm).
+Isso irá permitir que você troque versões do Node e até ter diferentes
+versões para cada projeto.
 
-### Install Blitz {#install-blitz}
+### Instale o Blitz {#install-blitz}
 
-Run `yarn global add blitz` or `npm install -g blitz --legacy-peer-deps`
-(legacy-peer-deps is needed because npm totally changed the behavior of peer
-deps and some dependencies haven't caught up)
+Rode `yarn global add blitz` ou `npm install -g blitz --legacy-peer-deps`
```

Comment:
Just a suggestion, I'd prefer to use 'Execute' instead of 'Rode'.

Reply:
Agreed